.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDatePeriodExtractor.cs

```diff
@@ -175,6 +179,20 @@ private static List<Token> GetTokenForRegexMatching(string text, Regex regex, Ex
             return ret;
         }
 
+        // Get token in after string
+        private static List<Token> GetTokenForRegexMatchingAfter(string text, Regex regex, ExtractResult er, int matchStart, int matchLength)
```

Comment:
I'm not sure I get why this method is written this way? Can you give examples of it and of the regular GetTokenForRegexMatching?
It seems to me both should be combined in some way.

Reply:
the two methods are now combined in GetTokenForRegexMatching